dataactcore/interfaces/function_bag.py

```diff
@@ -366,4 +366,25 @@ def get_submission_stats(submission_id):
         "total_obligations": float(base_query.scalar() or 0),
         "total_procurement_obligations": float(procurement.scalar() or 0),
         "total_assistance_obligations": float(fin_assist.scalar() or 0)
-    }
\ No newline at end of file
+    }
+
+
+def run_job_checks(job_id):
```

Comment:
No biggie, but the `run` here confuses me. We're only querying, not changing any sort of state.